testsuite/cluster-test/src/cluster_swarm/fullnode_spec_template.yaml

```diff
@@ -16,7 +16,7 @@ spec:
   nodeName: "{node_name}"
   initContainers:
   - name: init
-    image: 853397791086.dkr.ecr.us-west-2.amazonaws.com/libra_init:{image_tag}
+    image: 853397791086.dkr.ecr.us-west-2.amazonaws.com/libra_init:master
```

Comment:
Why did this change? Don't we still want to pick image tag?

Reply:
We are only using `kubectl` from the `libra_init` image and not any libra code.
Currently we are only building libra_validator and libra_cluster_test images during the build process and hence libra_init:{image_tag} is not available.

I was using `bitnami/kubectl:1.17.3` image earlier from dockerhub, but the problem with that is dockerhub rate limits us when we pull ~ 100 images at the same time.